src/components/BetaBanner/BetaBanner.jsx

```diff
@@ -2,13 +2,14 @@ import React from "react";
 import { useCookies } from "react-cookie";
 import { useTranslation } from "react-i18next";
 import { useDispatch } from "react-redux";
-import { CloseIcon } from "../../Icons";
+
 import { Link } from "react-router-dom";
 import Button from "../Button/Button";
-import { ExternalLinkIcon } from "../../Icons";
-import { showBetaModal } from "../Editor/EditorSlice";
+import ExternalLinkIcon from "../../assets/icons/external_link.svg";
+import { showBetaModal } from "../../redux/EditorSlice";
 
-import "./BetaBanner.scss";
+import "../../assets/stylesheets/BetaBanner.scss";
+import CloseIcon from "../../utils/CloseIcon";
```

Comment:
Just wondering but is there a reason to put this icon in utils as a component instead of adding it to icons like the others? Is it because of the scaling/translation requirements?

Reply:
Yes, I put it there because of the scaling prop. I think ideally we would put it in icons and scale it with CSS in the different scenarios where it needs scaling, or have small, medium and large versions of it, etc, but I decided to draw a line and call this "out of scope" to be honest with you.